Week1/homework/queries.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+// const util = require('util');
+const mysql = require('mysql');
+// const fs = require('fs');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser2',
+  password: 'hyfpassword',
+  database: 'world',
+});
+
+// const executeQuery = util.promisify(connection.query.bind(connection));
+
+function createQueries() {
+  // connection.connect();
+
+  // try {
+  //   await executeQuery(`SELECT country_name FROM countries WHERE population > 8000000`);
+  //   // await console.log(firstQuery);
+  //   await executeQuery(`SELECT country_name FROM countries WHERE country_name LIKE '%land%'`);
+  //   await executeQuery(
+  //     `SELECT city_name, population FROM cities WHERE population BETWEEN 500000 AND 1000000`
+  //   );
+  //   await executeQuery(`SELECT country_name FROM countries WHERE continent_name LIKE '%Europe%'`);
+  //   await executeQuery(`SELECT country_name FROM countries ORDER BY surface_area DESC`);
+  // } catch (error) {
+  //   console.log(error);
+  // }
+
+  connection.connect(error => {
```

Comment:
Would be nice if your application can get input from user and then execute the requested query. For example, if I type `1` from stdin, I expect question 1 in the homework.